test_runner/src/main/kotlin/ftl/util/Bash.kt

```diff
@@ -41,5 +41,5 @@ sealed class ShellEnvironment(val execution: String) {
     object Bash : ShellEnvironment("Bash.exe")
     object BinBash : ShellEnvironment("/bin/bash")
     object Cmd : ShellEnvironment("cmd.exe")
-    object Default : ShellEnvironment(if (isWindows) Cmd.execution else BinBash.execution)
+    object Default : ShellEnvironment(if (isWindows) "cmd.exe" else "/bin/bash")
```

Comment:
why this change?

Reply:
https://github.com/Flank/flank/runs/2928629408?check_suite_focus=true#step:6:339
It's related to the latest changes in the kotlin compiler. Locally, it will produce only warnings, but since our CI has `-Werror` flag enabled, the build would never finish successfuly

Reply:
@piotradamczyk5 @pawelpasterz we have a ticket for this: https://github.com/Flank/flank/issues/1770